www/components/ifdb-recommends.php

```diff
@@ -3,300 +3,9 @@
-                     from
-                       user_proximity
-                       join reviews as r1
-                       join games
-                       left outer join reviews as r2
-                         on games.id = r2.gameid and r2.userid = '$quid'
-                       left outer join playedgames as pg
-                         on games.id = pg.gameid and pg.userid = '$quid'
-                       left outer join wishlists as wl
-                         on games.id = wl.gameid and wl.userid = '$quid'
-                       left outer join unwishlists as uw
-                         on games.id = uw.gameid and uw.userid = '$quid'
-                     where
-                       r1.userid = user_proximity.userid
-                       and games.id = r1.gameid
-                       and r2.userid is null
-                       and pg.userid is null
-                       and wl.userid is null
-                       and uw.userid is null
-                       and r1.rating >= 4
-                       and ifnull(now() >= r1.embargodate, 1)
-                       and not (games.flags & " . FLAG_SHOULD_HIDE . ")
-                     group by
-                       r1.gameid
-                     order by
-                       dist
-                     limit
-                       0, $maxpicks", $db);
-
-                if ($debugflag) echo "error = " . mysql_error($db) . "<br>";
-
-                // fetch the results
-                for ($i = 0 ; $i < mysql_num_rows($result) ; $i++)
-                    $recs[] = mysql_fetch_array($result, MYSQL_ASSOC);
-
-                // the source of these recommendations is the collab filter #2
-                $recsrc = "collab2";
-
-                if ($debugflag) echo "record count=" . count($recs) . "<br>";
-            }
-        }
-    } else {
-      ?>
-      <div id="recommendations">Loading...</div>
-      <script nonce="<?php global $nonce; echo $nonce; ?>">
-        void function() {
-          var element = document.getElementById('recommendations')
-          var xhr = new XMLHttpRequest();
-          xhr.open("GET", '/async-recommendations<?= ($debugflag ? '?debug=yesDebug' : '')?>', true);
-          xhr.onreadystatechange = function() {
-            if (xhr.readyState !== 4) return;
-            if (xhr.status == 200) {
-              element.innerHTML = xhr.responseText;
-            } else {
-              element.innerHTML = "<span class=errmsg>There was an error loading recommendations.</span>";
-            }
-          }
-          xhr.send();
-        }();
-      </script>
-      <?php
-      return;
-    }
-}
-
-// if our recommendation list is empty, pick some random 4- and 5-star
-// games as recommendations
-if (count($recs) == 0) {
-
-    if ($debugflag) echo "falling back on default method<br>";
+// Pick some random 4- and 5-star games as recommendations
 
     // if we're logged in, don't pick games we've already rated
```

Comment:
Thanks! Let's also remove the indentation that was here for the `if` block.

Reply:
Ok, I think I got it.